api.go

```diff
@@ -88,7 +88,7 @@ func (c *Client) GetVersion(ctx context.Context) (string, error) {
 		return "", errNilReponse
 	case resp.StatusCode == http.StatusUnauthorized:
 		// Authenticate then try again
-		if _, err = c.Authenticate(ctx, c.configConnect); err != nil {
+		if _, err = c.Authenticate(ctx, c.configConnect); err == nil {
```

Comment:
**question:** Why the change? We would now be returning `nil`.

Reply:
checking, not sure why I did that. or was it not intended.